seacrowd/sea_datasets/indosmd/indosmd.py

```diff
@@ -221,7 +220,7 @@ def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
                             if key not in example["dialogue"][i]["data"]["slots"]:
                                 example["dialogue"][i]["data"]["slots"][key] = ""
 
-                if type(example["scenario"]["kb"]["items"]) == type(None):
+                if isinstance(example["scenario"]["kb"]["items"], None):
```

Comment:
Maybe there's an idiomatic way to do this rather than type-checking none?
You can check `None` types by:

```python
if example["scenario"]["kb"].get("items"):
    # do something
```

Reply:
Ah okay. That's working too. I've made the necessary revisions in the source code👍